project/Dependencies.scala

```diff
@@ -60,9 +63,11 @@ object Dependencies {
   }
 
   object PureConfig {
-    private val version = "0.17.1"
-    val pureconfig      = "com.github.pureconfig" %% "pureconfig"      % version
-    val cats            = "com.github.pureconfig" %% "pureconfig-cats" % version
+    private val version = "0.17.8"
+    val pureconfig      = "com.github.pureconfig" %% "pureconfig"             % version
+    val core            = "com.github.pureconfig" %% "pureconfig-core"        % version
+    val cats            = "com.github.pureconfig" %% "pureconfig-cats"        % version
+    val catsEffect      = "com.github.pureconfig" %% "pureconfig-cats-effect" % version
     val all             = Seq(pureconfig, cats)
```

Comment:
The val `all` no longer contains all dependencies listed in the object.

I think this is a temporary change to facilitate the migration to Scala 3. Maybe add TODOs to remind us to delete unused dependencies.

```suggestion
    private val version = "0.17.8"
    //TODO: Remove if still unused after Scala 3 migration complete
    val pureconfig      = "com.github.pureconfig" %% "pureconfig"             % version
    val core            = "com.github.pureconfig" %% "pureconfig-core"        % version
    //TODO: Remove if still unused after Scala 3 migration complete
    val cats            = "com.github.pureconfig" %% "pureconfig-cats"        % version
    val catsEffect      = "com.github.pureconfig" %% "pureconfig-cats-effect" % version
    val allScala2       = Seq(pureconfig, cats)
```

Reply:
Resolved in [ac57eaf](https://github.com/sky-uk/kafka-message-scheduler/pull/366/commits/ac57eaf40dec0701fa20d2c07a820113a2ad9df1)